src/steps/node.rs

```diff
@@ -91,11 +93,9 @@ impl NPM {
     fn upgrade(&self, run_type: RunType, use_sudo: bool) -> Result<()> {
         let args = ["update", self.global_location_arg()];
         if use_sudo {
-            run_type
-                .execute("sudo")
-                .arg(&self.command)
-                .args(args)
-                .status_checked()?;
+            let sudo_option = sudo();
+            let sudo = require_option(sudo_option, String::from("sudo is not installed"))?;
```

Comment:
We should see about making `require_option` take an `AsRef<str>` or `Display` or something.

Reply:
Good idea for the future.